src/core/achievements.cpp

```diff
@@ -3741,12 +3740,21 @@ static TinyString GetLoginEncryptionMachineKey()
 
   ret.resize(machine_guid_length);
   RegCloseKey(hKey);
-#elif !defined(__ANDROID__)
```

Comment:
this whole function is inside a `#ifndef __ANDROID__` block, so the extra condition here was redundant.